news/2023/2023-08-12-new-featured-artist-kardashev.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Kardashev"
+date: 2023-08-12 13:00:00 +0000
+---
+
+Your fingers quiver as **Kardashev** engulfs our Featured Artist library.
+
+![](https://assets.ppy.sh/artists/375/header.jpeg)
+
+Merging death metal with shoegaze (and a heap of other flavours), **Kardashev** has been outputting a distinctly atmospheric style of metal for over a decade. And as of today, that sound is officially part of osu!.
+
+**12** mind-altering tracks can be found on [**Kardashev**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/375), including a few beatmap templates that combine certain intro/outro tracks, like [*The Approaching Atonement* + *Silvered Shadows*](https://osu.ppy.sh/beatmapsets/1889320). Feel free to start mapping any of these insanely dense songs (at your own risk).
```

Comment:
going by https://kardashev.bandcamp.com/album/liminal-rite-deluxe

```suggestion
**12** mind-altering tracks can be found on [**Kardashev**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/375), including a few beatmap templates that combine certain intro/outro tracks, like [The Approaching of Atonement + Silvered Shadows](https://osu.ppy.sh/beatmapsets/1889320). Feel free to start mapping any of these insanely dense songs (at your own risk).
```

Reply:
This also looks correct by our metadata.